src/plugins/console/public/styles/_app.scss

```diff
@@ -22,6 +22,24 @@
   padding: 0 $euiSizeS;
 }
 
+.tourStepFixedContainer {
+  position: fixed; /* Fixed position ensures it stays in view */
```

Comment:
Could the same be accomplished with the `repositionOnScroll` prop on EuiTour?

Reply:
Nice catch! It's actually not needed at all! I initially added these stylings to specify a specific position of the second tour step, but I think its position is good as it is. I removed these new classes.